admin/pom.xml

```diff
@@ -59,7 +59,7 @@
         </dependency>
         <dependency>
             <groupId>org.openmrs.module</groupId>
-            <artifactId>emrapi-api-1.12</artifactId>
+            <artifactId>emrapi-api-2.2</artifactId>
```

Comment:
introduce a parameter for version-number? also is this is a specific artifcant .. would have expected something just "emrapi-api"

Reply:
@angshu We included emrapi-api along with above dependency. emrapi-api-2.2 needs to be included as the test cases are failing for "not being able to find the merged bean for org.openmrs.module.emrapi.diagnosis.DiagnosisService 
https://github.com/openmrs/openmrs-module-emrapi/blob/master/api-2.2/src/main/resources/moduleApplicationContext.xml#L30